.github/workflows/cd.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish
+
+on:
+  # Run on tags only
+  push:
+    tags:
+      - "*"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8"]
```

Comment:
why not 3.9 also?

Reply:
Ah ok! Got it, it has nothing to do with any targeted version on Pypi right?

Reply:
nothing